src/main/java/io/xpire/logic/commands/ViewCommand.java

```diff
@@ -10,13 +11,35 @@
 
     public static final String COMMAND_WORD = "view";
 
-    public static final String MESSAGE_SUCCESS = "Displayed all items";
+    public static final String MESSAGE_SUCCESS = "Displayed all items in %s list";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views all items in tracking or toReplenish list.\n"
+            + "If key is 'replenish', items in the toReplenish list will be displayed.\n"
+            + "If key is 'main', items in the main list will be displayed.\n"
+            + "Format: view|<key> (where key is 'main' or 'replenish')\n"
+            + "Example: " + COMMAND_WORD + "|main";
 
+    private ListToView list = null;
+
+    public ViewCommand() {
+    }
+
+    public ViewCommand(ListToView list) {
+        this.list = list;
+    }
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+        String output = String.format(MESSAGE_SUCCESS, "the");
```

Comment:
When will the message be "Displayed all items in the list"? Maybe just say "tracking list" it it's the xpireItem list?

Reply:
Currently, the message "Displayed all items in the list" is displayed when just "view" is called without a list argument, whereas a specific list will be mentioned if it is specified.

Reply:
So both lists will be displayed? Okay!